docs/speech_transcription/about.md

```diff
@@ -0,0 +1,59 @@
+---
+id: about
+title: Speech Transcription
+custom_edit_url: null
+---
+
+## Overview
+
+![Alt](100x100.png) The primary purpose of the application is to transcribe an audio sample into a written format that is accepted by SALT software. SALT software is an app that analyzes the speech of children and scores it according to their metrics. The app can currently transcribe audio samples of a conversation between a child and an adult into text. Children naturally produce speech sound errors, language errors, and language features, such as disfluencies. Language errors and features are diagnostically meaningful to speech-language pathologists (SLPs) and need to be maintained in the transcription. Unlike any programs currently available, the app will also offer suggestions for coding the sample.
```

Comment:
I suggest reordering this:
1. Start with "Children naturally produce speech sound errors, language errors, and language features, such as disfluencies. Language errors and features are diagnostically meaningful to speech-language pathologists (SLPs) and need to be maintained in the transcription."
2. Talk about SLPs using SALT software (mention that SALT is a commercially licensed software) to analyze speech and language errors.
3. Talk about what our application does.